src/common/types.cpp

```diff
@@ -400,7 +400,7 @@ string LogicalType::ToString() const {
 			if (i > 0) {
 				ret += ", ";
 			}
-			ret += "'" + KeywordHelper::WriteOptionallyQuoted(EnumType::GetString(*this, i).GetString(), '\'') + "'";
```

Comment:
I think this breaks round-tripping of ENUM types - is this change necessary in this PR?

Reply:
```
Query unexpectedly failed! (test/sql/export/parquet/export_parquet_enum.test:33)!
================================================================================
IMPORT DATABASE 'duckdb_unittest_tempdir/18323/export_test';
================================================================================
Parser Error: syntax error at or near "1"
LINE 5: CREATE TABLE tbl(a ENUM(''1'', ''2'', ''3''));
```

When I run the `export_parquet_enum.test` on `main`

Reply:
Since we already directly add singlequotes around it, I don't think it's necessary to add WriteOptionallyQuoted here, right?
@Mytherin

Reply:
Oh yeah you're right, I missed the explicit adding of quotes. I think we still need to deal with strings that contain `'` explicitly though. Could you add a test with an enum like this (`select 'hello''world'::enum('hello''world');`) - I think we should use `KeywordHelper::WriteQuotes` instead of adding the quotes